tests/datatypes/test_bytearray.py

```diff
@@ -19,6 +19,23 @@ def test_getattr(self):
             except AttributeError as err:
                 print(err)
             """)
+    def test_contains(self):
+        self.assertCodeExecution("""
+            print(bytearray([1,2,3]) in bytearray([1,2]))
+            print(bytearray([1,2]) in bytearray([1,2,3]))
+            print(bytearray([1,2,4]) in bytearray([1,2,3]))
+            print(bytearray([8,9,0,1]) in bytearray([1,2,3]))
+            print(101 in bytearray([1,2,3]))
+            print(101 in bytearray([1,2,3,101]))
+            print(b'pybee' in bytearray([1,2]))
+            print(bytearray([1,2]) in b'pybee')
+        """)
+        self.assertCodeExecution("""
+            print(300 in bytearray([1,2,3])) #Test ValueError invalid integer range
```

Comment:
The better way to test this is to use a try/catch block:
```
try:
    print(300 in bytearray([1,2,3]))
    print("No error raised")
except ValueError:
    print("Raised a ValueError")
```
This code will *always* run; but if a ValueError isn't raised, there will be a discrepancy in the output (which will be detected and reported)

Reply:
Changed that, thanks